tests/pytests/test_profile.py

```diff
@@ -574,6 +574,7 @@ def TimedOutWarningtestCoord(env):
     warning = coord_profile['Result processors profile']['profile']['Warning']
     env.assertEqual(warning, 'Timeout limit was reached')
 
+@skip(asan=True, msan=True)
```

Comment:
If this test is testing a unique flow, it might be better to just set a different number of docs, depending on what is the testing mode

Reply:
100%, but it's not so I prefer not to run it with sanitizers